container-images/intel-gpu/Containerfile

```diff
@@ -2,15 +2,17 @@ FROM quay.io/fedora/fedora:41 as builder
 
 COPY intel-gpu/oneAPI.repo /etc/yum.repos.d/
 
-RUN dnf install -y lspci clinfo intel-opencl g++ cmake git libcurl-devel intel-oneapi-base-toolkit ; \
-    git clone https://github.com/ggerganov/llama.cpp.git -b b4523 ; \
-    cd llama.cpp ; \
-    mkdir -p build ; \
-    cd build ; \
-    source /opt/intel/oneapi/setvars.sh ; \
-    cmake .. -DGGML_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_CURL=ON -DGGML_CCACHE=OFF -DGGML_NATIVE=ON ; \
-    cmake --build . --config Release -j -v ; \
-    cmake --install . --prefix /llama-cpp
+RUN dnf install -y lspci clinfo intel-opencl g++ cmake git libcurl-devel intel-oneapi-base-toolkit && \
+    git clone https://github.com/ggerganov/llama.cpp.git -b b4523 && \
+    cd llama.cpp && \
+    mkdir -p build && \
+    cd build && \
+    source /opt/intel/oneapi/setvars.sh && \
+    cmake .. -DGGML_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_CURL=ON -DGGML_CCACHE=OFF -DGGML_NATIVE=OFF && \
+    cmake --build . --config Release -j -v && \
+    cmake --install . --prefix /llama-cpp && \
+    cd ../.. && \
+    rm -rf llama.cpp
```

Comment:
Maybe this:

```
rm -rf llama.cpp
```

might help with space? 🤷 

Or maybe this layer is automatically deleted because it's not in the final image, I don't know.